src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius, String color) {
+        this.setColor(color);
+        this.radius = radius;
+        super.setType("circle");

```

Comment:
the type field is redundant in our case, because the name of the class describes the type